_posts/2022-05-04-#24584.md

```diff
@@ -10,15 +10,23 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+* One technique used for determining the payment address and amount is the ["Payment to different script type"](https://en.bitcoin.it/wiki/Privacy#Sending_to_a_different_script_type) heuristic.
+
+* [PR #23789](https://github.com/bitcoin/bitcoin/pull/23789) added payment address matching when generating a change address as a means of breaking the heuristic. This logic can lead to mixed UTXOs in the wallet. Depending on how these UTXOs are spent in the future, they might still leak information about which is the change/payment address in the original transaction.
+
+* This PR adds logic to avoid mixing different type UTXOs when funding a transaction
+
 ## Questions
 1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)?
--->
-
+2. In your own words, what is the "Payment to different script type" heuristic? How does it work?
+3. What are `OutputTypes` and `TxoutTypes`? How are they different?
```

Comment:
Please add links to the declarations to disambiguate what you're referring to + so that people know where to look for the answer when they are preparing for the meeting.